kyaml/fieldmeta/fieldmeta.go

```diff
@@ -151,10 +151,15 @@ func (fm *FieldMeta) Write(n *yaml.RNode) error {
 		delete(fm.Schema.VendorExtensible.Extensions, "x-kustomize")
 	}
 
-	// Ex: {"$ref":"#/definitions/io.k8s.cli.setters.replicas"} should be converted to
-	// {"openAPI":"replicas"} and added to the line comment
-	arr := strings.Split(fm.Schema.Ref.String(), ".")
-	n.YNode().LineComment = fmt.Sprintf(`{"%s":"%s"}`, shortHandRef, arr[len(arr)-1])
+	if fm.Schema.Ref.String() != "" {
```

Comment:
Can you explain the idea behind this in a comment.

Reply:
Done.